pallets/qa-tools/src/lib.rs

```diff
@@ -39,27 +41,41 @@ pub use pallet::*;
 mod tests;
 pub mod weights;
 pub use weights::*;
-mod pallet_tools;
+pub mod pallet_tools;
```

Comment:
Mix of `mod`, `pub mod`, `use` and `pub use` reduces readability. It would be better to group them somehow, for example:
```
pub mod
mod
pub use
use
```